crypto/bio/bio_lib.c

```diff
@@ -51,17 +50,17 @@ static long bio_call_callback(BIO *b, int oper, const char *argp, size_t len,
             return -1;
 
         argi = (int)len;
+    }
```

Comment:
The comment about ten lines up mentions _do nasty casts_ which the code isn't.  Worth a clean.